xchange-stream-ftx/src/main/java/info/bitrich/xchangestream/ftx/FtxStreamingAdapters.java

```diff
@@ -0,0 +1,94 @@
+package info.bitrich.xchangestream.ftx;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Streams;
+import info.bitrich.xchangestream.ftx.dto.FtxOrderbookResponse;
+import info.bitrich.xchangestream.service.netty.StreamingObjectMapperHelper;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.instrument.Instrument;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+import java.util.zip.CRC32;
+
+public class FtxStreamingAdapters {
+
+    private final static ObjectMapper mapper = StreamingObjectMapperHelper.getObjectMapper();
+
+    public static OrderBook adaptOrderbookMessage(OrderBook orderBook, Instrument instrument, JsonNode jsonNode){
+
+        Streams.stream(jsonNode)
+                .filter(JsonNode::isObject)
+                .map(res->{
+                    try{
+                        return mapper.readValue(res.toString(),FtxOrderbookResponse.class);
+                    }catch (IOException e){
+                        throw new RuntimeException(e);
+                    }
+                })
+                .forEach(
+                        message -> {
+
+                            if(message.getAction().equals("partial")){
+                                message.getAsks().forEach(ask->{
```

Comment:
lambda here and below doesn't need {} since it's single line

Reply:
done